src/pages/ReimbursementAccount/BeneficialOwnerInfo/BeneficialOwnerInfo.tsx

```diff
@@ -140,7 +122,11 @@ function BeneficialOwnerInfo({reimbursementAccount, reimbursementAccountDraft, o
         prevScreen,
         moveTo,
         resetScreenIndex,
-    } = useSubStep({bodyContent: BODY_CONTENT, startFrom, onFinished: handleBeneficialOwnerDetailsFormSubmit});
+    } = useSubStep<{beneficialOwnerBeingModifiedID: string; setBeneficialOwnerBeingModifiedID?: (id: string) => void}>({
```

Comment:
you are using this type in two places, it'd be more readable to extract it to its own type imo